README.md

```diff
@@ -7,19 +7,24 @@ following command:
 $ heroku create myapp --buildpack https://github.com/crystal-lang/heroku-buildpack-crystal.git
 ```
 
-In order for the buildpack to work properly you should have a `shard.yml`
-file, as it is how it will detect that your app is a Crystal app.
+The default behaviour is to use the [latest crystal release](https://github.com/crystal-lang/crystal/releases/latest).
+If you need to use a specific version create a `.crystal-version` file in your
+application root directory with the version that should be used (e.g. `0.17.1`).
 
-The default behaviour is to use the [latest crystal
-release](https://github.com/crystal-lang/crystal/releases/latest). If you need
-to use a specific version create a `.crystal-version` file in your application
-root directory with the version that should be used (e.g. `0.17.1`).
+## Requirements
 
-To learn more about how to deploy a Crystal application to Heroku, read [our
-blog post](http://crystal-lang.org/2016/05/26/heroku-buildpack.html).
+In order for the buildpack to work properly you should have a `shard.yml` file,
+as it is how it will detect that your app is a Crystal app.
 
-Your application has to listen on a port defined by Heroku.
-It is given to you through the command line option `--port` and the environment variable `PORT` (accessible through `ENV["PORT"]` in Crystal).
+Your application has to listen on a port defined by Heroku. It is given to you
+through the command line option `--port` and the environment variable `PORT`
+(accessible through `ENV["PORT"]` in Crystal). Usually you should not have to
+worry about this.
```

Comment:
I was thinking about when you are using a framework such as `Kemal` or `Frank`, in which case you don't really have to do anything, I believe. What would you need to worry about?